torch/nn/parallel/distributed.py

```diff
@@ -694,7 +694,11 @@ def __init__(
             )
 
         self.device_type = next(iter(distinct_device_types))
-
+        # use `torch.cuda` module to solve `xla` device
+        self._device_module_type = (
+            self.device_type if self.device_type != "xla" else "cuda"
```

Comment:
Can you explain this part
Cc @alanwaketan

Reply:
As torch.xla is not existed, this part added for some xla tests in CI flow while it seems that xla reuses cuda-related code in distributed.py.

Reply:
Maybe I can remove this part, and try to find the actual issue.

Reply:
@wconstab 
Could you help to trigger the ciflow?